include/z64.h

```diff
@@ -322,7 +322,8 @@ typedef enum {
 typedef struct {
     /* 0x000 */ char unk_00[0x128];
     /* 0x128 */ void* staticSegments[3];
-    /* 0x134 */ Gfx* dListBuf;
+    /* 0x134 */ union{ Gfx* dListBuf;
+                    Gfx (*dListBuf2)[150];};
```

Comment:
A union is not required if the code currently using `dListBuf` adapts to the `dListBuf2` type (use `&skyboxCtx->dListBuf2[n]` instead of `skyboxCtx->dListBuf + 150 * n`)

Reply:
Oh, yeah, that was just a hack to make the indices work out. I'll go back and fix that.